README.md

```diff
@@ -6,7 +6,13 @@ better alternative to that.
 
 ## Installation
 
-`go install github.com/segfaultax/go-check-graphite`
+1. Make sure you have Go installed first. If you're on a Mac, you can install
```

Comment:
These steps are redundant with every other go package, but since you took the effort let me clean it up slightly. What do you think of this?

---

1. [Install Go](https://golang.org/doc/install) (this project uses [Go Modules](https://blog.golang.org/using-go-modules), ensure you're using Go 1.11+)
1. Download: `go get github.com/segfaultax/go-check-graphite`
1. If you'd like to [install it locally](https://golang.org/cmd/go/#hdr-Compile_and_install_packages_and_dependencies): `go install github.com/segfaultax/go-check-graphite`

Note: When installed via `go install`, the executable will be named `go-check-graphite`.

Reply:
Sounds good to me! Thanks!